src/renderer/components/dex/MarketSelection.vue

```diff
@@ -21,32 +21,32 @@
         <div class="day-values">
           <div class="row">
             <div class="label">{{$t('vol')}}</div>
-            <div class="value">{{ $formatNumber($store.state.tradeHistory ? $store.state.tradeHistory.volume24Hour : 0) }}</div>
+            <div class="value">{{ $formatNumber(volume24Hour) }}</div>
```

Comment:
I like this pattern of removing the tertiary from the template and moving that logic into computed properties.